### DISCUSSION TITLE
Color swatch variables

### DISCUSSION THREAD

**Discussion by hoochalaffa** [Silly-V/Adobe-Illustrator#61] [Q&A]
Is there a way to swap out color swatches? I thought I read somewhere it was possible, but I couldn’t make sense of the instructions.

**Comment by schroef** [comment]
Could you be more specific?

You mean changing color in a document by others?

**Reply by hoochalaffa** [reply]
For example, creating a variable color swatch that is then called by the csv and replaced for each dataset. For example:

color1
PMS 123 C
PMS 289 C
PMS 1235 C

color1 would be the name of the color swatch to replace and the datasets would use swatches in the document to find and replace color1.

i don’t know how it works if doable via data merge. I thought I saw a supplementary script but I couldn’t find documentation on how to use it.

**Reply by schroef** [reply]
I believe he did describe that in the article on how to use it. Perhaps he could add links to this repo as well
https://www.linkedin.com/pulse/illustrator-variable-data-advanced-techniques-recoloring-vasily-hall/
